borg/hashindex.pyx

```diff
@@ -175,9 +192,39 @@ cdef class ChunkIndex(IndexBase):
 
     def __contains__(self, key):
         assert len(key) == self.key_size
-        data = <int *>hashindex_get(self.index, <char *>key)
+        data = <int32_t *>hashindex_get(self.index, <char *>key)
         return data != NULL
 
+    def incref(self, key):
+        """Increase refcount for 'key', return (refcount, size, csize)"""
+        assert len(key) == self.key_size
+        data = <int32_t *>hashindex_get(self.index, <char *>key)
+        if not data:
+            raise KeyError(key)
+        cdef int32_t refcount = _le32toh(data[0])
+        if refcount != INT32_MAX:
+            if INT32_MAX - refcount < 1:
```

Comment:
refcount is not equal to INT32_MAX (see previous "if"), thus INT32_MAX - refcount can never be 0.

So the "< 1" (which is equivalent to "<= 0" for integers) is misleading as it can never be 0.

So what you really meant here is "< 0"?

When is that the case?